### PR TITLE
Wip/setup django registration redux

### PR DESCRIPTION
**Pull request by bendavidsonku**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,9 @@ dev.db
 YourToDo/__pycache__/forms.cpython-35.pyc
```
Comment (JayOfferdahl) [inline .gitignore:2]: :+1:

File: YourToDo/settings.py [hunk 2]
```diff
@@ -230,6 +230,8 @@
     "mezzanine.forms",
     "mezzanine.galleries",
     "mezzanine.twitter",
+    'registration',
```
Comment (JayOfferdahl) [inline YourToDo/settings.py:235]: Include a comment above this to show what these are used for.

File: YourToDo/templates/registration/activation_complete.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
```
Comment (JayOfferdahl) [inline YourToDo/templates/registration/activation_complete.html]: Please include {% block title %}Registration Complete{% endblock%}

File: YourToDo/templates/registration/password_change_done.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{% extends "registration/registration_base.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Password changed" %}{% endblock %}
+
+{% block content %}
+<p>{% trans "Password successfully changed!" %}</p>
```
Comment (JayOfferdahl) [inline YourToDo/templates/registration/password_change_done.html:7]: Should we have a button to redirect users to the planner page or something else here?

File: YourToDo/urls.py [hunk 5]
```diff
@@ -35,15 +38,11 @@
...
-    # User Authentication Urls
-    url(r'^login/$',    'YourToDo.views.login'),
-    url(r'^logout/$',   lambda request: logout_then_login(request, "/"), name='logout'),
-
-    # User Registration Urls
-    url(r'^registration/$', 'YourToDo.views.registration'),
-    url(r'^registration_success/$', 'YourToDo.views.registration_success'),
     
+    # Django Registration Redux Urls
+    url(r'^accounts/logout/', 'YourToDo.views.ytdLogout', name='auth_logout'),
```
Comment (JayOfferdahl) [inline YourToDo/urls.py]: Any reason we called this ytdLogout? Where I come from, we actually use reasonable names. Please fix this immediately.

File: YourToDo/views.py [hunk 6]
```diff
@@ -39,44 +37,8 @@ def contact(request):
...
-
-    if request.method == 'POST':
-        form = defaultUserRegistrationForm(request.POST)
-        args['form'] = form
-        if form.is_valid():
-            form.save()
-            return HttpResponseRedirect('/registration_success/')
-    else:
-        args['form'] = defaultUserRegistrationForm()
+def ytdLogout(request):
```
Comment (JayOfferdahl) [inline YourToDo/views.py]: Again, change this name. Ridiculous.

File: YourToDo/templates/registration/password_change_done.html (hunk 4 above)
Comment (bendavidsonku) [inline YourToDo/templates/registration/password_change_done.html:7]: No fucktard

**Pull request merged**